packages/babel-plugin-proposal-object-rest-spread/test/fixtures/object-rest/duplicate-decl-bug/input.js

```diff
@@ -0,0 +1,4 @@
+it("es7.objectRestSpread", () => {
```

Comment:
hm, is this some kind of hidden feature of the test runner? or just a sample input that accidentally looks like a test? :)

Reply:
It's the `input.js` from the test I added, left over from reducing a real test case. The extra scope is necessary, though the exact string literal doesn't matter.

Reply:
i was wondering about `it`, good to know that it's "just" a sample input 👍